pom.xml

```diff
@@ -82,120 +82,103 @@
-						<mavenExecutorId>forked-path</mavenExecutorId>
-                        <pushChanges>false</pushChanges>
-					</configuration>
-				</plugin>
-
-
-				<plugin>
-					<artifactId>maven-deploy-plugin</artifactId>
-					<configuration>
-						<skip>false</skip>
-					</configuration>
-				</plugin>
-
-				<plugin>
-					<groupId>org.sonatype.plugins</groupId>
-					<artifactId>nexus-staging-maven-plugin</artifactId>
-					<version>1.6.8</version>
-					<extensions>true</extensions>
-					<executions>
-						<execution>
-							<id>default-deploy</id>
-							<phase>deploy</phase>
-							<goals>
-								<goal>deploy</goal>
-							</goals>
-						</execution>
-					</executions>
-					<configuration>
-						<serverId>sonatype-nexus-staging</serverId>
-						<nexusUrl>https://oss.sonatype.org/service/local/staging/deploy/maven2/</nexusUrl>
-					</configuration>
-				</plugin>
-				<plugin>
-    					<groupId>org.apache.maven.plugins</groupId>
-    					<artifactId>maven-javadoc-plugin</artifactId>
-					<configuration>
-						<author>false</author>
-        					<doctitle>Liquibase CDI ${project.version} API</doctitle>
-        					<quiet>true</quiet>
-        					<doclint>none</doclint>
-        					<encoding>UTF-8</encoding>
-        					<jarOutputDirectory>${project.basedir}/target</jarOutputDirectory>
-    					</configuration>
-    					<executions>
-        					<execution>
-            						<id>jar-javadoc</id>
-            						<goals>
-                						<goal>jar</goal>
-            						</goals>
-            						<phase>package</phase>
-        					</execution>
-    					</executions>
-				</plugin>
-			</plugins>
-
-		</pluginManagement>
+		<plugins>
+
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<configuration>
+					<encoding>UTF-8</encoding>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+					<optimize>true</optimize>
+					<debug>true</debug>
+					<encoding>${project.build.sourceEncoding}</encoding>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<skipTests>false</skipTests>
+					<redirectTestOutputToFile>true</redirectTestOutputToFile>
+					<reportFormat>plain</reportFormat>
+				</configuration>
+				<executions>
+					<execution>
+						<id>unit-tests</id>
+						<phase>test</phase>
+						<goals>
+							<goal>test</goal>
+						</goals>
+						<configuration>
+							<includes>
+								<include>**/*Test.java</include>
+							</includes>
+						</configuration>
+					</execution>
+					<execution>
+						<id>integration-tests</id>
+						<phase>integration-test</phase>
+						<goals>
+							<goal>test</goal>
+						</goals>
+						<configuration>
+							<includes>
+								<include>**/*IT.*</include>
+								<include>**/*Tests.*</include>
```

Comment:
Should this line be **/*Test.* and not Test**s**

Reply:
Good catch. This has been updated.